### ISSUE TITLE
Can't delete user

### ISSUE THREAD

**Issue by giuseppe99barchetta** [seerr-team/seerr#1042]
### Description

Can't delete a local Jellyseer user from the users page. 

"message": "Something went wrong deleting the user"

### Version

2.0.1

### Steps to Reproduce

1. Go to users page.
2. Try to delete a local user.
3. "message": "Something went wrong deleting the user"

### Screenshots

![Screenshot 2024-10-21 alle 18 03 08](https://github.com/user-attachments/assets/00600e16-13ac-4cc5-8223-a233f096a259)

### Logs

```shell
2024-10-21T16:03:04.418Z [error][API]: Something went wrong deleting a user{"errorMessage":"Cannot read properties of null (reading 'id')","userId":"3"}
```

### Platform

desktop

### Device

MacBook Pro M2

### Operating System

MacOs 15.1

### Browser

Safari

### Code of Conduct

- [X] I agree to follow Jellyseerr's Code of Conduct

**Comment by gauthier-th** [comment]
I can't reproduce this on a clean instance.

Can you reproduce this every time?

**Comment by giuseppe99barchetta** [comment]
> I can't reproduce this on a clean instance.
> 
> Anything specific on your instance ? Can you reproduce this every time?

Yeah, tried to reboot the docker instance and same error, also on chrome on Windows. Nothing particular about my instance, running docker inside portainer

**Comment by Fallenbagel** [comment]
> > I can't reproduce this on a clean instance.
> > 
> > Anything specific on your instance ? Can you reproduce this every time?
> 
> Yeah, tried to reboot the docker instance and same error, also on chrome on Windows. Nothing particular about my instance, running docker inside portainer

Does the error occur for every local user you create and when you try to delete them?

**Comment by giuseppe99barchetta** [comment]
> > > I can't reproduce this on a clean instance.
> > > Anything specific on your instance ? Can you reproduce this every time?
> > 
> > 
> > Yeah, tried to reboot the docker instance and same error, also on chrome on Windows. Nothing particular about my instance, running docker inside portainer
> 
> Does the error occur for every local user you create and when you try to delete them?

New user withouth any request history can be deleted normally. I'm trying to delete a user with 280 request in history

**Comment by Fallenbagel** [comment]
> > > > I can't reproduce this on a clean instance.
> > > > Anything specific on your instance ? Can you reproduce this every time?
> > > 
> > > 
> > > Yeah, tried to reboot the docker instance and same error, also on chrome on Windows. Nothing particular about my instance, running docker inside portainer
> > 
> > Does the error occur for every local user you create and when you try to delete them?
> 
> New user withouth any request history can be deleted normally. I'm trying to delete a user with 280 request in history

Might be a corrupted user

**Comment by giuseppe99barchetta** [comment]
> > > > > I can't reproduce this on a clean instance.
> > > > > Anything specific on your instance ? Can you reproduce this every time?
> > > > 
> > > > 
> > > > Yeah, tried to reboot the docker instance and same error, also on chrome on Windows. Nothing particular about my instance, running docker inside portainer
> > > 
> > > 
> > > Does the error occur for every local user you create and when you try to delete them?
> > 
> > 
> > New user withouth any request history can be deleted normally. I'm trying to delete a user with 280 request in history
> 
> Might be a corrupted user

How can I solve this? I want to delete all related requests

**Comment by Fallenbagel** [comment]
> Meanwhile, you could stop jellyseerr and edit db and delete user ID=3

**Comment by funkyfunked** [comment]
Had the exact same problem. You need to edit the file db.sqlite3 in your config/db folder and delete the data for the user. Just google how to edit a sqlite database with a gui for easy of use, edit the dq.sqlite3 with it, delete the user, start jellseerr again - all good again, all requests by this user are gone :)

**Comment by gauthier-th** [comment]
> Had the exact same problem.

Same error? Was that an user with a lot of requests too?

**Comment by funkyfunked** [comment]
> Same error? Was that an user with a lot of requests too?

Yes, a new jellyseerr user with about 400 movie requests within 2 days. After that i wanted to delete the user but was met with the error.

**Comment by gauthier-th** [comment]
Thanks. I'll take a look and see if I can replicate.

**Comment by badaz** [comment]
Hello, I just encountered the same bug.

For me it is a local user that I created to use with suggestarr. It did a dozen of requests which I all deleted. Now when I want to delete the user I'm getting `Something went wrong deleting a user {"userId":"8","errorMessage":"Cannot read properties of null (reading 'id')"}`

I'm running jellyseerr in docker on a Synology DS224+. I tried restarting the container and am getting the same error

**Comment by gauthier-th** [comment]
> EDIT 2 : from what I see [here in the code](https://github.com/Fallenbagel/jellyseerr/blob/develop/server/routes/user/index.ts#L410), user is found when first fetched otherwise the error message wouldn't know the user id. This is very strange indeed.

Not exactly, the error is displaying `req.params.id`, which is retrieved from the URL params from Express.
Without the full stack trace, I'm not sure to find the cause of this bug, because I don't see any reason for it to happen.
